src/styles/colors.ts

```diff
@@ -1,17 +1,28 @@
 export const COLORS = {
-    WHITE: '#FFF',
-    BLACK: '#000',
-    COD_GRAY: '#111111',
-    // SHARK_GRAY: '#232326',
-    VERY_DARK_GRAY: '#1F1F1F',
-    DARK_GRAY: '#2a2a2a',
-    TUNDORA_GRAY: '#4E4E4E',
-    GRAY: '#585858',
-    LIGHT_GRAY: '#9a9a9a',
+    WHITE: '#FFFFFF',
+    BLACK: '#000000',
+
+    AQUA: '#00DAFF', // main color
+    LIGHT_AQUA: '#00515E', // cards outline
+
+    WHITE_87: '#E0E0E0', // headers from page
+    WHITE_67: '#B1B1B1', // texts from page
+    WHITE_37: '#6A6A6A', // texts from fields
+
+    GRAY_CARDS: '#28282A', // all cards
```

Comment:
color names should not be based on features. you can use https://www.color-blindness.com/color-name-hue/ to get names for different kind of colors